src/aatoolbox/utils/codab.py

```diff
@@ -0,0 +1,43 @@
+"""Retrieve COD administrative boundaries."""
+import geopandas as gpd
+
+from aatoolbox.config.aatoolbox import AaToolbox
+
+MODULE_BASE_DIR = "cod_ab"
+
+
+class CodAB(AaToolbox):
+    """
+    Work with COD AB (administrative boundaries).
+
+    Parameters
+    ----------
+    iso3 : (str)
+        country iso3
+    """
+
+    def __init__(self, iso3: str):
+        super().__init__(iso3=iso3, module_base_dir=MODULE_BASE_DIR)
+
+    def get_admin0(self):
+        """
+        Get the admin level 0 COD AB for a country.
+
+        Returns
+        -------
+        geopandas dataframe with COD AB admin 0
+
+        Examples
+        --------
+        >>> from aatoolbox.utils.codab import CodAB
+        >>> # Get admin 0 boundaries for Nepal
+        >>> codab = CodAB("npl")
+        >>> npl_admin0 = codab.get_admin0()
+        """
+        shapefile = (
+            self.get_public_raw_dir()
+            / self.config.country.codab.base_dir
+            / f"{self.config.country.codab.base_zip}"
```

Comment:
Thinking out loud, is it too much to require it to be a zip file? At least for most countries I didn't use the zip but unpacked it. Not sure what is the most common approach, thinking about users outside our team

Reply:
I think we should keep it as whatever format it is downloaded as from HDX, which I think is often a zip, as ideally we want to automate the downloading part as well

Reply:
But we will have to refactor this function in the future to accomodate the possible file types